src/chrome/content/rules/tagesschau.de.xml

```diff
@@ -1,52 +1,53 @@
 <!--
+	tageschau.de refuse https
+	
 	Non-functional subdomains:
-		- atlas		(connection refused)
-
-	¹ Hostname mismatch
-
-	Insecure cookies are set for these domains:
-
-		- www.tagesschau.de
+		atlas (connection refused)
 
+	Redirect to Flypsite.com
+		duell.live.tagesschau.de
+		
+	Redirect to tagesschau.de
+		tagesschau24.de
+		www.tagesschau24.de
 
 	Mixed content:
-
-		- Images, on:
-
-			- wetter and www from wetter *
-			- wetter from www *
-
-		- Bugs, on:
-
-			- wetter and www from de.sitestat.com *
-			- wetter and www from gsea.ivwbox.de *
-			- wetter and www from tagessch.ivwbox.de *
-			- www from ts.flyp.tv *
-
-	* Secured by us
+		karten.tagesschau.de
+		wahl.tagesschau.de
+		
+	Invalid Certificate-Chain:
+		todo.tagesschau.de
+		
+	Invalid Certificate:
+		blog.tagesschau.de
+		origin.blog.tagesschau.de
+		intern.tagesschau.de
+		korrespondenten.tagesschau.de
+		stat.tagesschau.de
+		rbb24.wahl.tagesschau.de
+		wahlarchiv.tagesschau.de
 
 -->
 <ruleset name="tagesschau.de (partial)">
-	<target host=       "tagesschau.de"/>
-	<target host=    "www.tagesschau.de"/>
-	<target host= "karten.tagesschau.de"/>
-	<target host=  "media.tagesschau.de"/>
-	<target host=   "meta.tagesschau.de"/>
-	<target host=   "wahl.tagesschau.de"/>
-	<target host= "wetter.tagesschau.de"/>
-
-
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^www\.tagesschau\.de$" name="X-Mapping-[a-z]+$" /-->
+	<target host="tagesschau.de"/>
+	<target host="www.tagesschau.de"/>
+	<target host="faktenfinder.tagesschau.de"/>
+	<target host="media.tagesschau.de"/>
+	<target host="download.media.tagesschau.de"/>
+	<target host="karten.tageschau.de"/>
+	<target host="meta.tagesschau.de"/>
+	<target host="prodforum.tagesschau.de"/>
+	<target host="service.tagesschau.de"/>
+	<target host="wahl.tagesschau.de"/>
+	<target host="wetter.tagesschau.de"/>
+	<target host="programm.tagesschau24.de"/>
 
 	<securecookie host="^www\.tagesschau\.de$" name=".+" />
 
-	<rule from="^http://tagesschau\.de/"
+    <rule from="^http://tagesschau\.de/"
```

Comment:
Please convert spaces to tabs and remove trailing tabs (around `Invalid Certificate-Chain` block)